blocks/hero/hero.js

```diff
@@ -19,8 +19,9 @@ export default async function decorate(block) {
       { class: 'grid-container' },
       div(
         { class: 'title column' },
-        h1(
-          heroProperties.Title,
+        p(
```

Comment:
the title should still be an h1. This is the first heading on the page and to be consistent and correct w/ accessibility and h1 should exist.